src/app/styles/css/mixins/variables.css

```diff
@@ -57,6 +57,7 @@
 
   /* https://about.twitter.com/content/dam/about-twitter/company/brand-resources/en_us/Twitter_Brand_Guidelines_V2_0.pdf */
   --twitterBlue: #1da1f2;
+  --xBlack: #000;
```

Comment:
yes you read that correctly, i checked their logo, it's straight up raw `#000` color, designers around the world rolling their eyes